index.src.html

```diff
@@ -504,64 +491,28 @@
+## Credential Attestation Statement (interface <dfn interface>WebAuthnAttestation</dfn>) ## {#iface-attestation}
 
-Authenticators also provide some form of key attestation. The basic requirement is that the authenticator can produce, for each
+Authenticators also provide some form of attestation. The basic requirement is that the authenticator can produce, for each
 credential public key, attestation information that can be verified by a <a>[RP]</a>. Typically this information contains a
 signature by an attesting key over the attested public key and a challenge, as well as a certificate or similar information
 providing provenance information for the attesting key, enabling a trust decision to be made.
 
-<div dfn-for="AttestationStatement">
-    The <dfn>header</dfn> element contains the attestation header, including algorithm, (optionally) the claimed AAGUID and
-    (optionally) the attestation certificate chain.
-
-    The <dfn>core</dfn> element describes the data that is being attested to, and its type.
-
-    The <dfn>signature</dfn> element contains the base64url-encoded attestation signature. The structure of this object depends
-    on the signature algorithm specified in the header.
-</div>
-
-This attestation statement is delivered to the <a>[RP]</a> according to the Client API. It contains all the information that the
-[RP]'s server requires to reconstruct the signature base string, as well as to decode and validate the bindings of both the
-client- and authenticator data.
-
-
-## Credential Attestation Header (interface <dfn interface>AttestationHeader</dfn>) ## {#sec-attestation-header}
-
-This structure contains additional data required to verify the attestation signature.
+<div dfn-for="WebAuthnAttestation">
+    The <dfn>type</dfn> member specifies the type of attestation statement contained in this structure. This specification
+    defines a number of attestation types, in [[#defined-attestation-types]]. Other attestation types may be defined in later
+    versions of this specification.
 
-<div dfn-for="AttestationHeader">
-    The <dfn>claimedAAGUID</dfn> element contains the claimed Authenticator Attestation GUID (a version 4 GUID, see
-    [[RFC4122]]). This value is used by the <a>[RP]</a> to look up the trust anchor for verifying the following
-    {{AttestationCore}} object. If the verification succeeds, the AAGUID related to the trust anchor is trusted. This field MUST
-    be present, if either no attestation certificates are used (e.g., DAA) or if the attestation certificate doesn't contain the
-    AAGUID value in an extension.
-
-    The <dfn>x5c</dfn> attribute contains the attestation certificate and its certificate chain as described in [[!RFC7515]]
-    section 4.1.6.
+    The <dfn>clientData</dfn> member contains the base64url encoding of the parameters sent to the authenticator by the client.
```

Comment:
It's sent to the authenticator by the client. The client uses the RP-supplied parameters as input to its decision, sure, but what the client decides to send is what gets signed over, and the RP only has an indirect influence over that.